README.rst

```diff
@@ -15,6 +15,8 @@ the network).
 
 Installation
 ------------
+Argcomplete requires Python 2.7 or 3.3+.
```

Comment:
Please move this to the bottom of the readme, next to the *Common Problems* section.

Space at the top of the readme is precious and we shouldn't clutter it with warnings about python versions that roughly nobody uses.